### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by balmainn**
Code Review

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+default: main.c rectangle.c part1.go
+	go build part1.go
```
Comment (jacobGolden) [inline Makefile:2]: not totally sure on this, but i think default needs to have part1 and part2, not the rectangle program. Still stumped on the makefile in general tbh

**Review by jacobGolden — COMMENTED** [review]
looks mostly good, just need to get that sweet sweet makefile lol

File: Makefile (hunk 1 above)
Comment (balmainn) [reply Makefile:2]: okay i can fix that makes sense to me

**Comment by niedermeb** [comment]
I like the panic condition in part1 and the clean target in the makefile. Maybe you could review the instructions for part1.

**Comment by balmainn** [comment]
fixed makefile bug and everything should be working now. Merging.

**Pull request merged**